src/objects/container/container.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import basicDemo from './demo/basic.twig';
+import './demo/styles.scss';
+import './container.scss';
+
+<Meta title="Objects/Container" decorators={[withKnobs]} />
+
+# Container
+
+<Preview>
+  <Story
+    name="Basic"
+    height="300px"
+    parameters={{ paddings: { disabled: true } }}
+  >
+    {basicDemo({
+      prose: boolean('Add o-container--prose modifier to the container', false),
+    })}
+  </Story>
+</Preview>
+
+## Prose
+
+<Preview>
+  <Story
+    name="Prose"
+    height="300px"
+    parameters={{ paddings: { disabled: true } }}
+  >
+    {basicDemo({
+      prose: boolean('Add o-container--prose modifier to the container', true),
+    })}
+  </Story>
+</Preview>
+
+## Themes
+
+<Preview>
+  <Story
+    name="Themes (Dark)"
+    height="300px"
+    parameters={{ themes: { disable: true }, paddings: { disabled: true } }}
+  >
+    {basicDemo({
+      prose: boolean('Add o-container--prose modifier to the container', false),
+      theme: 'dark',
+    })}
+  </Story>
+</Preview>
```

Comment:
It doesn't look like the container has any theme-specific styles, so it's probably overkill to include this here. One can always opt into the dark theme from the canvas view if they want to test that it works there.